uberpoet/filegen.py

```diff
@@ -30,18 +30,25 @@
 }}"""
 
 swift_class_template = """
-public class MyClass{0} {{
+@objc
```

Comment:
this is needed if we were to create this class from ObjC->Swift.

Reply:
Makes sense